benefits/eligibility/forms.py

```diff
@@ -30,8 +30,11 @@ def __init__(self, agency: models.TransitAgency, *args, **kwargs):
         self.classes = "col-lg-8"
         # second element is not used since we render the whole label using selection_label_template,
         # therefore set to None
-        self.fields["verifier"].choices = [(v.id, None) for v in verifiers]
-        self.fields["verifier"].widget.selection_label_templates = {v.id: v.selection_label_template for v in verifiers}
+        verifier_field = self.fields["verifier"]
+        verifier_field.choices = [(v.id, None) for v in verifiers]
+        verifier_field.widget.selection_label_templates = {v.id: v.selection_label_template for v in verifiers}
+        verifier_field.widget.attrs.update({"data-custom-validity": _("Please choose a transit benefit.")})
```

Comment:
Oops, I don't think this string `"Please choose a transit benefit."` made its way into the `.po` files?

Reply:
Good catch, thanks -- I added it in new commit https://github.com/cal-itp/benefits/pull/2045/commits/ead179bb898a118e6cca7f03adad742bd65499c0